lib/identity_key/hw_unique_key_internal.h

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Is this file a duplicate?

Reply:
Yep, and only the printing was being used as far as I can see. So I removed it and just added some printing macros.